cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -116,26 +116,26 @@ public BuildCommand(Path userDir, PrintStream outStream, PrintStream errStream,
     @CommandLine.Option(names = {"--all", "-a"}, description = "Build or compile all the modules of the project.")
     private boolean buildAll;
     
-    @CommandLine.Option(names = {"--output", "-o"}, description = "Writes output to the given file. The provided " +
-                                                                  "output filename may or may not contain the '.jar' " +
+    @CommandLine.Option(names = {"--output", "-o"}, description = "Write the output to the given file. The provided " +
```

Comment:
"Writes" is correct in this context isn't it? This is basically saying "The build commands writest the output..."

Reply:
We use "write" as a convention following the practice used in docs of similar communities like [1].

[1] https://golang.org/cmd/go/